cypress-tests/cypress/e2e/PaymentUtils/Xendit.js

```diff
@@ -0,0 +1,624 @@
+          billing: billingDetails,
+        },
+        currency: "IDR",
+        mandate_data: singleUseMandateData,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    "3DSManualCapture": {
+      Request: {
+        amount: 6500000,
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "IDR",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_customer_action",
+          setup_future_usage: "on_session",
+          payment_method_data: paymentMethodData3ds,
+        },
+      },
+    },
+    MandateSingleUseNo3DSManualCapture: {
+      Configs: {
+        DELAY: {
+          STATUS: true,
+          TIMEOUT: 3000,
+        },
+      },
+      Request: {
+        payment_method: "card",
+        amount: 6500000,
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+          billing: billingDetails,
+        },
+        currency: "IDR",
+        mandate_data: singleUseMandateData,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "processing",
+        },
+      },
+    },
+    Capture: {
+      Configs: {
+        DELAY: {
+          STATUS: true,
+          TIMEOUT: 3000,
+        },
+      },
+      Request: {
+        payment_method: "card",
+        amount: 6500000,
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "IDR",
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+          amount: 6500000,
+          amount_capturable: 0,
+          amount_received: 6500000,
+        },
+      },
+    },
+    PartialCapture: {
+      Configs: {
+        DELAY: {
+          STATUS: true,
+          TIMEOUT: 3000,
+        },
+      },
+      Request: {
+        amount: 6500000,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "partially_captured",
+          amount: 6500000,
+          amount_capturable: 0,
+          amount_received: 100,
+        },
+      },
+    },
+    Refund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "IDR",
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    VoidAfterConfirm: {
+      Request: {},
+      Response: {
+        status: 400,
+        body: {
+          error: {
+            type: "invalid_request",
+            message: "Cancel/Void flow is not supported",
+            code: "IR_19",
+          },
+        },
+      },
+    },
```

Comment:
@likhinbopanna, we should take this up in the coming days and put this under getCustomExchange